sklearn/cluster/optics_.py

```diff
@@ -516,9 +525,9 @@ def _set_reach_dist(self, point_index, processed, X, nbrs):
         self.reachability_[unproc[improved]] = rdists[improved]
         self.predecessor_[unproc[improved]] = point_index
 
-        # Define return order based on reachability distance
-        return (unproc[quick_scan(np.take(self.reachability_, unproc),
-                                  dists)])
+        # Choose next based on smallest reachability distance
```

Comment:
Maybe add a comment like this : if there're multiple points with the same reachability distance, then we ...